cylc/flow/scripts/lint.py

```diff
@@ -125,11 +125,35 @@
         'url': STYLE_GUIDE + 'trailing-whitespace',
         'index': 6
     },
-    re.compile(r'inherit\s*=\s*[a-z].*$'): {
+    # Look for families both from inherit=FAMILY and FAMILY:trigger-all/any
+    re.compile(r'(inherit\s*=\s*.*[a-z].*)|(\w[a-z]\w:\w+?-a(ll|ny))'): {
         'short': 'Family name contains lowercase characters.',
         'url': STYLE_GUIDE + 'task-naming-conventions',
         'index': 7
     },
+    re.compile(r'{[{%]'): {
+        'short': JINJA2_FOUND_WITHOUT_SHEBANG,
+        'url': '',
+        'index': 8
+    },
```

Comment:
This is not working for me. I don't get any warning when using Jinja2 syntax without a shebang line

Reply:
Can you post your workflow definition? I can't replicate this.

Reply:
Ah dang, never mind. The workflow I was testing on had a leftover `pyproject.toml` ignoring this rule!